src/components/m-datepicker/index.js

```diff
@@ -24,13 +24,13 @@ class AXADatepicker extends BaseComponentGlobal {
     classes: PropTypes.string,
     buttonOk: PropTypes.string,
     buttonCancel: PropTypes.string,
-    locale: localePropType,
+    locale: PropTypes.string,
     value: PropTypes.string,
     startYear: startType,
     startMonth: startType, // zero-based
-    selectedDay: PropTypes.number,
-    lowerEndYear: PropTypes.number,
-    higherEndYear: PropTypes.number,
+    selectedDay: PropTypes.string, // TODO check type
```

Comment:
Could make sense to allow for `valuePropType`, i.d. `number` and `string`

Reply:
I removed all datepicker changes. they were not meant to be in this PR.